griptape/memory/structure/conversation_memory.py

```diff
@@ -16,6 +16,11 @@ class ConversationMemory:
     driver: Optional[BaseConversationMemoryDriver] = field(default=None, kw_only=True)
     runs: list[Run] = field(factory=list, kw_only=True)
     structure: Structure = field(init=False)
+    autoload: bool = field(default=True, kw_only=True)
+
+    def __attrs_post_init__(self) -> None:
+        if self.driver and self.autoload:
+            [self.add_run(r) for r in self.driver.load().runs]
```

Comment:
Perhaps we should change the return type `load()` to `Optional[ConversationMemory]` and check here if it's `None` or not.